documentation/modules/con-kafka-logging.adoc

```diff
@@ -0,0 +1,89 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+[id='con-kafka-logging-{context}']
+= Kafka loggers
+
+Kafka has its own configurable loggers:
+
+* `kafka.root.logger.level`
+* `log4j.logger.org.I0Itec.zkclient.ZkClient`
+* `log4j.logger.org.apache.zookeeper`
+* `log4j.logger.kafka`
+* `log4j.logger.org.apache.kafka`
+* `log4j.logger.kafka.request.logger`
+* `log4j.logger.kafka.network.Processor`
+* `log4j.logger.kafka.server.KafkaApis`
+* `log4j.logger.kafka.network.RequestChannel$`
+* `log4j.logger.kafka.controller`
+* `log4j.logger.kafka.log.LogCleaner`
+* `log4j.logger.state.change.logger`
+* `log4j.logger.kafka.authorizer.logger`
+
+Kafka and ZooKeeper use the Apache `log4j` logger implementation.
+
+Use the `logging` property to configure loggers and logger levels.
+
+You can set the log levels by specifying the logger and level directly (inline) or use a custom (external) ConfigMap.
+If a ConfigMap is used, it must be placed under under the `log4j.properties` key according.
```

Comment:
ditto as above

Reply:
Repeated word: "under".

Reply:
Accordingly? Or perhaps remove the word.

Reply:
Fixed as above